incubator/elasticsearch/templates/configmap.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
 data:
   elasticsearch.yml: |-
     cluster.name: {{ .Values.cluster.name }}
-    
+
     node.data: ${NODE_DATA:true}
     node.master: ${NODE_MASTER:true}
 {{- if hasPrefix "5." .Values.appVersion }}
```

Comment:
I believe this is the only real issue right now. App version is now 6.x. Fixing this line and the same thing below should gets tests to pass, I believe.

Reply:
I am fixing this in #5663